src/pages/ProductListing.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import {useEffect} from "react"
+import Navbar from '../Components/Navbar'
+import axios from 'axios';
+import FilterBar from '../Components/FilterBar'
+import ProductCard from '../Components/ProductCard'
+function ProductListing() {
+  const [products, setProducts] = React.useState([]);
+  const getData = async () => {
+     const data = await axios.get('/api/products');
+    try{
+      console.log(data.data.products);
+      setProducts(data.data.products);
+    }
+    catch{
+      console.log("error")
```

Comment:
Remove console statements before deployment.
You can use throw(error)

Reply:
Yes, I'll remove them!